### PR TITLE
Added search to the Kubeflow website.

### PR DESCRIPTION
**Pull request by sarahmaddox**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/471)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Fixes https://github.com/kubeflow/website/issues/335

File: config.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ pygmentsStyle = "tango"
  #images = ["images/project-illustration.png"]
 
  # Docsy: Google Custom Search Engine ID. Remove or comment out to disable search.
-#gcs_engine_id = "011737558837375720776:fsdu1nryfng"
+gcs_engine_id = "0007239566369470735695:624rglujm-w"
```
Comment (LisaFC) [inline config.toml]: Move this line (and its comment) down to the [params] section (under # Site params) or Hugo won't pick it up as a site parameter.

Comment (sarahmaddox) [reply config.toml]: Thanks @LisaFC !

**Comment by sarahmaddox** [comment]
Preview https://deploy-preview-471--competent-brattain-de2d6d.netlify.com/docs/

The search currently returns no results. I'm guessing that's because this is a preview, and thus not on www.kubeflow.org

I've removed the "Getting Started" link from the top navigation bar, because it's getting quite cluttered up there.

File: resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.content [hunk 2]
```diff
@@ -27,7 +27,7 @@ limitations under the License.
                     }
 
                     var query = $(this).val();
-                    var searchPage = "/search/?q=" + query;
+                    var searchPage = "//localhost:1313/search/?q=" + query;
```
Comment (abhi-g) [inline resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.content]: the reference to localhost looks like you might have made to test locally. Is this added to the PR in error?

Comment (sarahmaddox) [reply resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.content]: Yes! Good catch. I thought I'd reverted that but it snuck through.

I'm putting a hold on this PR until I see if I can improve the search results too. But please do continue with the review, especially the UI changes.
/hold

**Comment by sarahmaddox** [comment]
I'm going to try building our site with the env variable set to production, to index the site. Evidently we've never done that. Could help our SEO as well as fix the search box.

**Comment by sarahmaddox** [comment]
@abhi-g Our SEO is looking better since we set the Hugo environment to production. I think the search box should show some results too, once it's on kubeflow.org. Let's go ahead and push this PR.

**Comment by abhi-g** [comment]
Awesome.

/lgtm
/approve

**Pull request merged**